dashboard/app/helpers/levels_helper.rb

```diff
@@ -570,6 +570,9 @@ def blockly_options
     level_options['puzzle_number'] = script_level ? script_level.position : 1
     level_options['lesson_total'] = script_level ? script_level.lesson_total : 1
     level_options['final_level'] = script_level.final_level? if script_level
+    level_options['lastLevelInLesson'] = script_level.end_of_lesson? if script_level
```

Comment:
nit: we're switching between snake- and camel-case here, we should stay consistent

Reply:
There is a mix of snake_case and camelCase in appOptions and it looked like there were more camelCase properties. 🤷‍♀️  I'm in favor of consistency, but didn't know which to be consistent with.  
![Screen Shot 2022-01-07 at 3 48 29 PM](https://user-images.githubusercontent.com/12300669/148605507-74d8bb2b-ba42-436d-ac79-173150b2becc.png)

Reply:
ahh understandable. let's keep as-is -- since these values are just for the frontend and we aren't doing any snake-to-camel-case transformation anywhere, these should follow JS convention